legacy/src/api/merchants/merchants.md

```diff
@@ -271,6 +271,52 @@ Returns a [paginated][] list of Merchants attached to an Account.
 }
 {% endjson %}
 
+### Search Merchants
+
+{% reqspec %}
+  GET '/api/merchants/search'
+  auth 'api-key'
+  query_param 'origin', '123.42,32.22'
+  query_param 'distance', '100'
+  query_param 'name', 'Store'
+{% endreqspec %}
+
+{% h4 Request Query Parameters %}
+
+|       Query Parameter   |           Type            |                                 Description                                                         |
+| :---------------------- | :------------------------ | :--------------------------------------------------------------------------------------------------- |
+| {% break _ origin %}    | {% dt String %} {% opt %} | The point to be searched around, formatted as a latitude,longitude                                   |
+| {% break _ distance %}  | {% dt Number %} {% opt %} | The distance from the origin to be searched around in km                                             |
+| {% break _ assetType %} | {% dt String %} {% opt %} | The [AssetType][] to filter merchants by. This can be supplied multiple times to further filter results  |
+| {% break _ name %}      | {% dt String %} {% opt %} | Name of the merchant                                                                                 |
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+	items: [
+		{
+			"id": "M001",
+			"name": "Store 1",
+			"acceptedAssetTypes": [
```

Comment:
@bpal938 This response payload uses a new model that we are not documented anywhere